addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -129,6 +129,8 @@ function factory(dependencies) {
                         partner_id,
                         partner_name,
                     });
+                case 'mass_mailing':
```

Comment:
As I mentioned in an earlier comment, we will want to sync other kind of information this way too in the future, so I think we should directly make this a generic notification for receiving updated "channel_info".

In this case the handler can be made generic, taking all arbitrary channel fields that are forwarded to `convertData`.

I'm thinking the best API for the generic notification payload should be:
```python3
{
    'type': 'channel_info',
    # ... here any other meta data that we would like to share, for example the context of the change if necessary
    'channel': {
        # ... here any field that we want to update, ideally following the format they have in `channel_info()`
    },
}
```

See also task-1891151.

Reply:
@seb-odoo . just have a look. Thanks